Framework/PythonInterface/plugins/algorithms/LoadLamp.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import (absolute_import, division, print_function)
+from mantid.api import FileProperty, WorkspaceProperty, PythonAlgorithm, NumericAxis, AlgorithmFactory, FileAction
+from mantid.kernel import Direction
+from mantid.simpleapi import CreateWorkspace, AddSampleLogMultiple, mtd
+
+import numpy
+import h5py
+
+
+class LoadLamp(PythonAlgorithm):
```

Comment:
Could Lamp ever use another data format? Possibly `LoadLampHDF` might be more descriptive?

Reply:
Lamp can export other formats, but they are ASCII, which we publicly do not support in Mantid. So I think adding HDF5 to the algorithm name is a bit of an overkill. The documentation and summary should make it clear enough.